src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -824,9 +830,9 @@ export default class ConvasController {
         cc.save();
 
         if (!aircraft.inside_ctr) {
-            cc.fillStyle = this.theme.RADAR_TARGET_OUTSIDE_RANGE;
+            cc.fillStyle = this.theme.HISTORY_DOT_OUTSIDE_RANGE;
```

Comment:
I'd really love to see us get rid of some of these `else`es by doing something like:
```javascript
let fillStyle = this.theme.HISTORY_DOT_INSIDE_RANGE;

if (!aircraft.inside_ctr) {
  fillStyle = this.theme.HISTORY_DOT_OUTSIDE_RANGE;
}

cc.fillStyle = fillStyle;
```